db/db_test2.cc

```diff
@@ -1084,6 +1088,9 @@ TEST_F(DBTest2, PresetCompressionDict) {
           options.compression_opts.max_dict_bytes = kBlockSizeBytes;
           options.compression_opts.zstd_max_train_bytes =
               kZstdTrainFactor * kBlockSizeBytes;
+          options.bottommost_compression_opts.max_dict_bytes = kBlockSizeBytes;
```

Comment:
are these changes needed? It looks like we're not setting `enabled=true` here so the values in `bottommost_compression_opts` shouldn't be used, right?

Reply:
Yeah.....I will remove them